src/components/RemoteConfig/index.tsx

```diff
@@ -41,21 +41,34 @@ function RemoteConfig() {
     useTemplate();
 
   const [searchText, setSearchText] = useState('');
+  const [edited, setEdited] = useState(false);
   const [paramBeingEdited, editParam] = useState<string | undefined>(undefined);
   const [editTemplate, setEditTemplate] = useState(
     JSON.parse(JSON.stringify(template))
   );
 
-  const editing = isEqual(template, editTemplate);
+  const editing = isEqual(template, editTemplate) && !edited;
+
+  useEffect(() => {
```

Comment:
please add a comment to make it clear what this is doing (reset the edit template when a new template comes in from the server?)

Reply:
should this be a `useMemo` instead of a `useEffect`?